apiary-metastore-events/kafka-metastore-events/kafka-events-common/src/main/java/com/expediagroup/apiary/extensions/events/metastore/common/event/SerializableListenerEvent.java

```diff
@@ -0,0 +1,101 @@
+/**
+ * Copyright (C) 2018-2019 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expediagroup.apiary.extensions.events.metastore.common.event;
+
+import java.io.Serializable;
+import java.util.Map;
+import java.util.Objects;
+
+import javax.annotation.concurrent.NotThreadSafe;
+
+import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
+import org.apache.hadoop.hive.metastore.api.EnvironmentContext;
+import org.apache.hadoop.hive.metastore.events.ListenerEvent;
+
+/**
+ * @see {@link ListenerEvent}
+ */
+@NotThreadSafe
```

Comment:
Why do have this?

Reply:
Possibly because the `ListenerEvent` class also has this annotation? Can remove if you think its not needed

Reply:
If we can't think why it's needed then I'd remove it.